packages/lingua-franca/src/components/display/GreyButton.tsx

```diff
@@ -1,33 +0,0 @@
-import * as React from "react"
-import { withPrefix } from "gatsby"
-
-export type Props = {
```

Comment:
Well, this branch is titled `spring-cleaning` for a reason. `GreyButton` might be useful, but does it bring you joy?

It did not bring me joy because code for grey buttons and the like is not so hard to come by (on the Internet). But I will not second-guess what brings joy to you... **`GreyButton` shall stay.**

Reply:
I'm also fine deleting it. My general take is that library components that are well-tuned to the website already might be worth keeping around for future use. Let's purge components that do not qualify as such.

Reply:
Yes, we are of the same mind. I brought back most of these deleted files, and the only one that I would really prefer not to keep is `invertCodeToHTML`, since that does not sound like something that we will ever need (nor does it sound like a good idea, for if we have the original code, we need not parse HTML in order to recover it).

Reply:
Awesome. What remained to be said: this clean up sure does bring me joy!